src/test/kotlin/org/rust/lang/core/completion/RsKeywordCompletionContributorTest.kt

```diff
@@ -589,6 +589,30 @@ class RsKeywordCompletionContributorTest : RsCompletionTestBase() {
         "fn foo<T /*caret*/>() {}"
     """)
 
+    fun `test const inside trait`() = checkCompletion("const", """
```

Comment:
Let's add a test to check that we don't suggest `enum`, `trait`, etc keywords inside `trait` & `impl` blocks